build/dsc/templates/dsc_type.rb.erb

```diff
@@ -1,4 +1,5 @@
 require 'pathname'
+require File.expand_path(File.join(File.dirname(__FILE__), '../../puppet_x/puppetlabs/dsc_type_helpers'))
```

Comment:
Is there a compelling reason for `'../..' + 'puppet/_x/puppetlabs/dsc_type_helpers'` vs `'../../puppet/_x/puppetlabs/dsc_type_helpers'`?

In other words, is that extra string concat there for some good reason?